pkg/flow/adapter/transformation/adapter.go

```diff
@@ -83,21 +87,21 @@ func NewAdapter(ctx context.Context, envAcc pkgadapter.EnvConfigAccessor, ceClie
 	trnContext, trnData := []v1alpha1.Transform{}, []v1alpha1.Transform{}
 	err := json.Unmarshal([]byte(env.TransformationContext), &trnContext)
 	if err != nil {
-		log.Fatalf("Cannot unmarshal context transformation env variable: %v", err)
+		logger.Fatalf("Cannot unmarshal context transformation env variable: %v", err)
```

Comment:
In other components we often put the actual Go error in a separate field using

```go
logger.Panicw("What happened in clear text", zap.Error(err))
```

so the log ends up looking like

```json
{
  "msg": "What happened in clear text",
  "err": "the detail of the error"
}
```

It's not mandatory, just a suggestion to remain as consistent as possible.

Reply:
Yeah, this is a common issue across all our logs, I'll handle it in a separate PR